angular-spa-ui/js/admin-app/content-manager-module/contentManager-ctrl.js

```diff
@@ -1,90 +1,104 @@
 "use strict";
 
-module.exports = function ($scope, $state, promises, $stateParams, cmService, configService, contentStorage) {
+module.exports = function ($scope, $state, promises, $stateParams, cmService, configService, contentStorage, rlService) {
+
+    var vm = this;
```

Comment:
vm = "view model", that in general it looks quite logical. This is connected to controllerAs synax and use controllers without $scope. 
When we use controllerAs syntax, all not private variables is connected to view and it is logical to call them "view model".
As an option, it can be "_this" or "that",